code/stata-before-saving.do

```diff
@@ -1,18 +1,22 @@
-* If the data set has ID variables, create a local and test
-* if they are fully and uniquely identifying the observations.
-local idvars household_ID household_member year
-isid `idvars'
+* If the data set has ID variables, test if they uniquely identifying the observations.
+
+    local idvars household_ID household_member year
+    isid `idvars'
 
 * Sort and order on the idvars (or any other variables if there are no ID variables)
-sort  `idvars'
-order * , seq // Place all variables in alphanumeric order (optional but useful)
-order `idvars' , first // Make sure the idvars are the leftmost vars when browsing
+
+    sort  `idvars'
+    order * , seq // Place all variables in alphanumeric order (optional but useful)
+    order `idvars' , first // Make sure the idvars are the leftmost vars when browsing
 
 * Drop intermediate variables no longer needed
 
 * Optimize disk space
-compress
 
-* Save data settings
-save "${myProject}/myDataFile.dta" , replace // The folder global is set in master do-file
- use "${myProject}/myDataFile.dta" , clear // It is useful to be able to recall the data quickly
+    compress
+
+* Save data
+
+    save "${myProject}/myDataFile.dta" , replace // The folder global is set in master do-file
+    saveold "${myProject}/myDataFile-13.dta" , replace v(13) // For others
+     use "${myProject}/myDataFile.dta" , clear // It is useful to be able to recall the data quickly
```

Comment:
Not sure I understand why the data is being loaded right after saving if it's already open.

Reply:
Eh it's personal preference, and it's useful interactively. I agree it's unusual and can be removed, but it saves a lot of keystrokes